.github/workflows/rundoc-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: rundoc-ci-test
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  test-rundoc:
+    runs-on: pub-hk-ubuntu-24.04-ip
```

Comment:
This repo is public, why not use one of the public GHA runners like `ubuntu-latest`?